Building an API.md

```diff
@@ -0,0 +1,53 @@
+# Intro
+
+Building an API is different than building a regular HTML application. There are
+no HTML views to worry about, but instead you should take extra care about status responses, response format and especially documentation since your application will be used
+by other developers.
+
+## Creating an application
+
+To generate a Rails application with only the elements necessary for an API
+application use the `--api` flag e.g.
+`rails new my_application --api`
```

Comment:
This may be true, but you will need an admin panel for 99% of applications, so the --api flag wont do